services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/csv_download_for_progress_report.jsx

```diff
@@ -75,7 +80,8 @@ export default class CSVDownloadForProgressReports extends React.Component {
   }
 
   closeDropdownIfOpen(e) {
-    if (this.state.showDropdown && e.target.classList.value !== 'btn button-green' && e.target.classList.value !== 'print-button' && e.target.classList.value !== 'print-img') {
+    const ignoreClasses = [printBtnClass, printImgClass, clickedDownloadBtnClass, unclickedDownloadBtnClass]
```

Comment:
I think we probably still also want to check for `this.props.className` in here in case one is getting passed down?

Reply:
Good call.  That seems reasonable.